draft-wang-ppm-differential-privacy.md

```diff
@@ -451,9 +465,81 @@ class DpMechanism:
 
 > TODO: Specify a Gaussian sampler
 
-## Randomized Response {#rr}
+## Binary Randomized Response {#rr}
+
+The binary randomized response mechanism specified here aims to provide the
+DP guarantee at the Client level. It is the same as the one specified in
+Appendix C.1 of {{MJTB+22}}.
+
+To apply noise at Client side, the Client flips a coin at each coordinate of its
+vector. If the coin turns out to be a head, the Client will change the value `x`
```

Comment:
We need a paragraph before this describing what type of data the mechanism is compatible with. Giving an illustrative example would help to.